<launch>
    <node name="qnx2ros" pkg="message_transformer" type="qnx2ros" output="screen">
        <param name="server_port" value="43897" />   
        <param name="is_imu_pub" value="true" />
    </node>
    <node name="ros2qnx" pkg="message_transformer" type="ros2qnx" output="screen">
        <param name="vel_x_factor" value="1.0" />
        <param name="local_port" value="43894"/>
        <param name="remote_port" value="43893"/>
        <param name="remote_ip" value="192.168.1.120"/>
        <param name="isdebug" value="false"/>
    </node>
    <node name="nx2app" pkg="message_transformer" type="nx2app" output="screen" >
        <param name="server_port" value="43899" />
    </node>
    <node name="sensor_checker" pkg="message_transformer" type="sensor_checker" output="screen">
        <param name="imu_silent_limit" value="3"/>
        <param name="odom_silent_limit" value="3"/>
        <param name="odom2_silent_limit" value="3"/>
        <param name="joint_silent_limit" value="3"/>
        <param name="realsense_silent_time" value="3"/>
        <param name="lidar_silent_time" value="3"/>
        <param name="ultrasound_silent_time" value="3"/>
        <param name="check_freq_hz" value="5"/>
        <param name="isdebug" value="false"/>
    </node>
</launch>
